btc/path_transaction.go

```diff
@@ -47,6 +51,19 @@ func (b *backend) pathTransactionWrite(ctx context.Context, req *logical.Request
 	if multisig {
 		walletName = MultiSigPrefix + walletName
 	}
+	t := d.Get("token").(string)
+	if t == "" {
+		return nil, errors.New(MissingTokenError)
+	}
+
+	// check if auth token is valid
+	token, err := b.GetToken(ctx, req.Storage, t)
+	if err != nil {
+		return nil, err
+	}
+	if token == nil {
+		return nil, errors.New(InvalidTokenError)
+	}
 
```

Comment:
In this way remain valid for an hour, signing multiple times. we should revoke it as soon we return the signature

Reply:
so, fix it as you can, given the current architecture. you can "Revoke all tokens by accessor"

Reply:
Update: now tokens are revoked after the operation is successfully completed and have a TTL of 5 minutes.